### PR TITLE
move run-app to run module

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
move run-app to run module

### CODE REVIEW

**Comment by kakaZhou719** [comment]
pls wait registry , this PR just do some code moving actions.

File: cmd/sealer/cmd/cluster/run.go [hunk 1]
```diff
@@ -137,7 +157,7 @@ func NewRunCmd() *cobra.Command {
 	runCmd.Flags().StringSliceVar(&runFlags.CMDArgs, "cmd-args", []string{}, "set args for image cmd instruction")
 	runCmd.Flags().StringSliceVarP(&runFlags.CustomEnv, "env", "e", []string{}, "set custom environment variables")
 	runCmd.Flags().StringVarP(&clusterFile, "Clusterfile", "f", "", "Clusterfile path to run a Kubernetes cluster")
-
+	runCmd.Flags().StringSliceVar(&runFlags.LaunchCmds, "cmds", []string{}, "override default LaunchCmds of clusterimage")
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/run.go:160]: what will happen if I pass this flag for non-app image

Comment (Stevent-fei) [reply cmd/sealer/cmd/cluster/run.go:164]: > 

I added the error message prompt
image

File: cmd/sealer/cmd/cluster/run.go [hunk 2]
```diff
@@ -73,6 +77,43 @@ func NewRunCmd() *cobra.Command {
...
+				return err
+			}
+
+			extension, err := imageEngine.GetSealerImageExtension(&imagecommon.GetImageAnnoOptions{ImageNameOrID: args[0]})
+			if err != nil {
+				return fmt.Errorf("failed to get cluster image extension: %s", err)
+			}
+
+			if extension.Type == v12.AppInstaller {
+				logrus.Infof("start to install application: %s", args[0])
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/run.go:91]: the args[0] is not a application name, how about use  `start to install app image` here

File: cmd/sealer/cmd/cluster/run.go [hunk 3]
```diff
@@ -73,6 +77,43 @@ func NewRunCmd() *cobra.Command {
...
+					Image:      args[0],
+					Platform:   "local",
+				}); err != nil {
+					return err
+				}
+				return installApplication(args[0], nil, extension, infraDriver, imageEngine, applyMode)
+			}
+
+			if len(runFlags.LaunchCmds) > 0 {
+				return fmt.Errorf("this command parameter (--cmds) is only applicable to application images")
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/run.go:115]: s/applicable/available

File: cmd/sealer/cmd/cluster/run.go [hunk 4]
```diff
@@ -73,6 +77,43 @@ func NewRunCmd() *cobra.Command {
...
+
+				if err := imageEngine.Pull(&imagecommon.PullOptions{
+					Quiet:      false,
+					PullPolicy: "missing",
+					Image:      args[0],
+					Platform:   "local",
+				}); err != nil {
+					return err
+				}
+				return installApplication(args[0], nil, extension, infraDriver, imageEngine, applyMode)
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/run.go:111]: not need to pass nil if you have added the error message prompt for non-app image with `LaunchCmds`

**Review by Starnop — APPROVED** [review]

**Pull request merged**